arches/app/frontend/webpack/webpack.common.js

```diff
@@ -209,8 +221,8 @@ module.exports = {
                 type: 'javascript/auto',
             },
             {
-                test: /\.htm$/i,
-                loader: "html-loader",
+                test: /\.html?$/i,
+                use: ["html-loader"],
```

Comment:
what's up with this?

Reply:
It was just a test of html loader functionality; but it allows for .htm or .html extensions to be used.  There is a .html extension - registration/base.html, so this should probably stay - or the file extension should be changed.  I am also not sure how this will play out with arches subprojects in the future, so the lookback may be needed.

Reply:
great! ty